qiita_db/study.py

```diff
@@ -347,6 +348,56 @@ def create(cls, owner, title, efo, info, investigation=None):
 
         return cls(study_id)
 
+    @classmethod
+    def delete(cls, id_):
+        r"""Deletes the study from the database
+
+        Parameters
+        ----------
+        id_ : integer
+            The object identifier
+
+        Raises
+        ------
+        QiitaDBError
+            If the sample_(id_) table exists means a sample template exists
+        """
+        cls._check_subclass()
+
```

Comment:
See https://github.com/biocore/qiita/issues/1078. For the time being, I added a instantiation of `Study(id_)` to validate this case and a test just to be sure.